tests/components/config/test_config_entries.py

```diff
@@ -287,7 +287,7 @@ async def async_step_user(self, user_input=None):
     }
 
 
-async def test_create_account(hass, client):
+async def test_create_account(hass, client, enable_custom_integrations):
```

Comment:
Why do we need to enable custom integrations ?

Reply:
Because "test" is a custom integration <https://github.com/home-assistant/core/tree/dev/tests/testing_config/custom_components/test>

Reply:
Ah. How come this worked before ?

Reply:
That's a really good question! will look into that

Reply:
Before we used Integration.resolve_legacy(hass, domain) which used _lookup_path as fallback which then included custom integrations.
Now we have stricter loading rules

Reply:
aah check!